frontend/src/components/Navigation.js

```diff
@@ -118,8 +115,8 @@ const NavigationAuth = () => {
                         {idToken && <img className={classes.navIconSize} crossorigin="anonymous" src={`http://localhost:5000/users/picture?idToken=${idToken}`} alt='profile' />}
                         </button>
                         <div className='dropdownContent'>
-                            <div className={classes.dropdownLink}><a className={classes.link} href='/profile'>Profile</a></div>
-                            <div className={classes.dropdownLink}><a className={classes.link} href='/logout'>Logout</a></div>
+                            <div className={classes.dropdownLink}><a className={classes.link} onMouseEnter={changeColorMouseEnter} onMouseLeave={changeColorMouseLeave} href='/profile'>Profile</a></div>
+                            <div className={classes.dropdownLink}><a className={classes.link} onMouseEnter={changeColorMouseEnter} onMouseLeave={changeColorMouseLeave} onClick={doLogOut}>Logout</a></div>
```

Comment:
When I logout from a private page, you get hit with a 404 error message, aside from that the logout functionality works. We should redirect the user to the home page or something when they hit the logout button so that they don't see this message. I think a simple `<Redirect to {`/`} />` would work here